### ISSUE TITLE
Feature: Support hostname in addition to IP

### ISSUE THREAD

**Issue by mstberto** [bitfocus/companion-module-renewedvision-propresenter#46]
Not sure if there was something preventing this, but it would be great to be able to use a hostname as opposed to IP. Our network has a search domain so we can reference computers by a local hostname (prodpc.domainname.com). We can assign static IPs, but prefer to use hostnames when possible in case those change.

**Comment by greyshirtguy** [comment]
Right now, it is configured to only accept IP addresses....
I havent properly tested, but I suspect the module could be easily updated to also accept and use hostnames. 
I'll check this soon and have a think to make sure enabling of hostnames won't lead to any unexpected issues.

Please note - I would recommend you consider allocating static IP for any device on your production network.
There is an industry stadard practice that says "If you need constant access to a device, or server, in the network it would be beneficial for that address not to change".

It's a little more work to setup a static address pool (or Manual DHCP assignments) on your network and then manually manage which devices get one of those static IP addresses. But it is generally considered best practice because of the universally accepted principal that "any changes, to anything" generally leads to an increases risk of isssues and failures.

**Comment by mstberto** [comment]
We do! We just reorganized the network to support better isolation which meant I had to redo a good chunk of static IPs. I’ve reduced what doesn’t require a static IP which made it easier, but it’s a lot easier remembering a host name than static IPs when you have a lot of subnets.

The computers I’ll be connecting do have static IPs, but I’d love to just enter the host name cause it’s easier to remember and I wouldn’t have to update the IP with it changing the rare times it may. 😅
